README.rst

```diff
@@ -173,18 +187,9 @@ SoCo supports the following controls amongst others:
    -  MAC Address
 
 -  Set the speaker’s Zone Name
--  Find all the Sonos speakers in a network.
--  Put all Sonos speakers in a network into “party mode”.
--  “Unjoin” speakers from a group.
--  Manage the Sonos queue (get the items in it, add to it, clear it,
-   play a specific song from it)
--  Get the saved favorite radio stations and shows (title and stream
-   URI)
--  Search for and play item from your music library
 -  Start a music library update and determine if one is in progress
 
-SoCo also supports lower level access from Python to all Sonos services (eg
-Alarms)
+SoCo also supports lower level access from Python to all Sonos services.
```

Comment:
I actually think it was ok with an example here, but maybe better with something like: `... (e.g. ContentDirectory or RenderingControl)

Reply:
Good idea.